Jenkinsfile

```diff
@@ -7,7 +7,7 @@ stage('Trust') {
     enforceTrustedApproval()
 }
 
-fhBuildNode {
+fhBuildNode([labels: ['nodejs6']]) {
```

Comment:
@camilamacedo86 you have changed from an object to a function for fhBuildNode, Any way of testing this has breaking changes ?

Reply:
@austincunningham should just force a node6 build node to be used, like [here](https://github.com/fheng/fh-supercore/blob/master/Jenkinsfile#L6) in other components

@camilamacedo86 node6 is out of LTS, should we use node8 at least here? (presuming we have node8 build nodes..)

Reply:
No, it should stay at nodejs6 like it is here, since that's the version we run the platform components on. If we move to Node.js 10 for the platform, that's when this would go to nodejs10.